opentelemetry-sdk/tests/logs/test_export.py

```diff
@@ -175,7 +175,7 @@ def test_simple_log_record_processor_shutdown(self):
 
     def test_simple_log_record_processor_different_msg_types(self):
         exporter = InMemoryLogExporter()
-        log_record_processor = BatchLogRecordProcessor(exporter)
+        log_record_processor = SimpleLogRecordProcessor(exporter)
```

Comment:
This fixes an existing test that wasn't failing but was testing the wrong processor.